_sass/icons.scss

```diff
@@ -0,0 +1,92 @@
+.candlestick-icon{
+	background-image: svg-url('<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 50 50"><defs><style>.cls-1{fill:none;}.cls-1,.cls-2{stroke:#{rgba($primaryBrandColor,1)};stroke-miterlimit:10;stroke-width:2px;}.cls-2{fill:#{rgba($contentInverseColor,1)};}</style></defs><title>candle</title><line class="cls-1" x1="25" y1="8.5" x2="25" y2="41.5"/><rect class="cls-2" x="20.842" y="17.146" width="8.316" height="15.708"/></svg>');
```

Comment:
@4p00rv 
- In order to avoid browser compatibility issues, please remove `<defs>` and its contents from all svg codes. The style can be moved to tags as attributes instead of classes.
- Some attributes/tags like `id`, `data-name`, `<title>` can be removed as well.
- Please also include `width` and `height` attributes.

A good example for above items is what you have in `.chart_template-icon`